### PR TITLE
YG-08: Add datetimepicker, remove list of doctor form, update code sdk

### PR DESCRIPTION
**Pull request by phienngo**
YG-08: Add datetimepicker, remove list of doctor form, update code sdk

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,18 @@
...
-    script = document.createElement('script');
-    script.async = true;
-    script.src =  '//sdk.medkumo.loc:' + window.location.port + '/medkumo.js';
-    entry = document.getElementsByTagName('script')[0];
-    entry.parentNode.insertBefore(script, entry);
-  })();
-  window.Medkumo_ready = function() {
-    Medkumo.init('<your hospital key>');
-  };
+  function myFunction() {
```
Comment (tailehuu) [inline README.md]: remove this

File: README.md [hunk 2]
```diff
@@ -1,21 +1,18 @@
...
+    Medkumo.showBookAnAppointment('<hospital_key>', '<doctor_key>')
+  }
 </script>
 ```
-
+After that you can be bind click event to the function.
+Example:
+```html
+<button onclick="myFunction()">Booking</button>
```
Comment (tailehuu) [inline README.md]: how about this ?

... onclick="Medkumo.showBookAnAppointment('<hospital_key>', '<doctor_key>')" ...

File: README.md (hunk 1 above)
Comment (phienngo) [reply README.md]: Ok TaiLe,
I will remove this and update README.md file.

The updated README.md file:
You must insert the source code below at the bottom of the body tag to render the Medkudo Book An Appointment.

<script src='//sdk.medkumo.loc:85/medkumo.js'></script>
After that you can be bind click event to the function. Example:

<button onclick="Medkumo.showBookAnAppointment('<hospital_key>', '<doctor_key>')">Booking</button>
Note
src: is the source file public of medkumo.js, ex: http://sdk.medkumo.loc:85/medkumo.js

File: README.md (hunk 2 above)
Comment (phienngo) [reply README.md]: Ok TaiLe.
I will update this.

**Pull request merged**